addons/web/static/src/js/views/basic/basic_renderer.js

```diff
@@ -49,7 +49,7 @@ var BasicRenderer = AbstractRenderer.extend({
             if (!canBeSaved) {
                 invalidFields.push(widget.name);
             }
-            widget.$el.toggleClass('o_field_invalid', !canBeSaved);
+            $(widget.$el).toggleClass('o_field_invalid', !canBeSaved);
```

Comment:
I'd rather wrap the instruction in a if (widget.el) or something (in master, we toggle 2 classnames like this)

Reply:
and I'd add a comment // widget may not be started yet